### PR TITLE
Site Design Wizard: Theme Mod Controls

### PR DESCRIPTION
**Pull request by jeffersonrabb**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Adds a 2nd tab to the Site Design Wizard with the Theme Options.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Navigate to Site Design > Settings
3. Change the different options and check if they are being applied.
4. Check the Customizer if the 2 matches.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/wizards/site-design/views/theme-mods/index.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	/**
+	 * Render.
+	 */
+	render() {
+		const { themeMods, setThemeMods } = this.props;
+		const {
+			header_solid_background: headerSolidBackground,
+			header_simplified: headerSimplified,
+			header_center_logo: headerCenterLogo,
+			show_author_bio: authorBio,
```
Comment (adekbadek) [inline assets/wizards/site-design/views/theme-mods/index.js]:
```suggestion
			show_author_bio: authorBio = true,
```
This is one of the two mods which have non-false defaults. Until it's set (here or in the customiser) it's value won't show up in the API response.

Comment (jeffersonrabb) [reply assets/wizards/site-design/views/theme-mods/index.js]: Fixed in https://github.com/Automattic/newspack-plugin/pull/433/commits/00401ab41b9918d55a0e2d436ebcd0f505365b1c

**Comment by jeffersonrabb** [comment]
> plus some [tiny linting nitpicks](https://app.circleci.com/jobs/github/Automattic/newspack-plugin/199/parallel-runs/0/steps/0-103)

Linting errors fixed in https://github.com/Automattic/newspack-plugin/pull/433/commits/545ddb0288fa812b5690d2a837e60beba83f6110

**Comment by jeffersonrabb** [comment]
@thomasguillot do you think this screen should have a preview feature?

**Comment by thomasguillot** [comment]
> @thomasguillot do you think this screen should have a preview feature?

@jeffersonrabb the "Advanced settings" button currently acts as a preview since it takes you to the customiser.

I think once the Site Design is fully complete (all settings moved to the wizard) I will consider removing the customiser button in favour of a preview.

**Pull request merged**